cookbook/core/containerization/raw_container.py

```diff
@@ -193,3 +194,25 @@ def wf(a: float, b: float):
 # ^^^^^^^^^^^^^^^^^^^^^^^^^
 # .. literalinclude::  ../../../../core/containerization/raw-containers-supporting-files/per-language/julia/calculate-ellipse-area.jl
 #    :language: julia
+
+# %%
+# Use Raw Container in the map task
+# =================================
+#
+# To use raw container in the map task, users have to
+# 1. Manually parse the input data. The input is string representation of a list, so we need to convert it to python list.
+# 2. Get array index (BATCH_JOB_ARRAY_INDEX_VAR_NAME), and use it to get the element in the list.
+def main(a: str, output_dir):
+    # convert a to list
+    res = eval(a)
```

Comment:
what is eval(a)?  is this a `[]` literal?  could we get an example of it?  is there any other way of getting the list object like loading it through json.loads or something?

Reply:
yeah maybe we don't use `eval` in our examples... or at least `json.loads`